docs/main/updating/plugins/4-0.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Updating plugins to 4.0
+description: Guide for updating a Capacitor v3 plugin to v4
+slug: /updating/plugins/4-0
+---
+
+# Updating a Capacitor v3 plugin to 4.0
+
+There are only a few changes required to update a plugin from Capacitor 3 to Capacitor 4.
+
+:::info
+While you can follow the manual steps below, check out [this tool](https://github.com/rdlabo-team/capacitor-plugin-to-v4) from community member [@rdlabo](https://twitter.com/rdlabo) that automatically updates a Capacitor 3 plugin to v4.
+:::
+
+## Android
+
+### Update Android Project Variables
+
+In your `build.gradle` file, update your values to the following new minimums:
+
+```groovy
```

Comment:
this block is confusing, looks like you copied the changes from the variables.gradle file, but in plugins there is no variables.gradle file and the values are in different places and with a different format (using `:` instead of `=` )
So maybe instead of a code block use a list saying
minSdkVersion to 22
compileSdkVersion to 32
etc.